packages/angular-playground/cli/src/serve-angular-cli.ts

```diff
@@ -21,24 +21,19 @@ export async function serveAngularCli(config: Config) {
     return Promise.resolve();
 }
 
-function configureArguments(config: Config): string[] {
-    let args = [config.angularCliPath, 'serve', '-no-progress'];
-
+function configureArguments(config) {
+    let args = [config.angularCliPath, 'serve', '--no-progress', '--aot'];
```

Comment:
aot since otherwise angular-cli / webpack does not complain about missing chunk for dynamic imports.